src/Psalm/Internal/Analyzer/Statements/Expression/ArrayAnalyzer.php

```diff
@@ -220,7 +224,8 @@ private static function analyzeArrayItem(
         Context $context,
         ArrayCreationInfo $array_creation_info,
         int $int_offset,
-        PhpParser\Node\Expr\ArrayItem $item
+        PhpParser\Node\Expr\ArrayItem $item,
+        Codebase $codebase = null
```

Comment:
this is never not null

Reply:
I was thinking of backwards compatibility for changing the method signature but I see it's private now.

Reply:
Fixed